### ISSUE TITLE
Free version of V2 colab runs out of RAM

### ISSUE THREAD

**Issue by jorymo** [TheLastBen/fast-stable-diffusion#690]
Not sure if this is something that can  be fixed yet, but when I use the version for free colab, it still runs out of RAM when trying to launch

**Comment by Maxxxwell62** [comment]
Same here on free ver.

LatentDiffusion: Running in v-prediction mode
DiffusionWrapper has 865.91 M params.
Downloading: 100% 3.94G/3.94G [02:24<00:00, 27.4MB/s]
^C

**Comment by elithecho** [comment]
Yep getting this too, falling back to v1.5 for now

**Comment by TheLastBen** [comment]
Did you use the VRAM_FIX ?

**Comment by elithecho** [comment]
Thanks @TheLastBen ,
I'm using this one, should I be using the other on instead?

https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111-V2.ipynb

**Comment by TheLastBen** [comment]
@elithecho this colab is for the V2 model for users using free colab, it doesn't use gdrive (to save RAM somehow).

**Comment by jorymo** [comment]
> Did you use the VRAM_FIX ?

Yeah, that's what made the link to the V2 Colab pop up, if that makes sense. I didn't mount my Google Drive, but it still runs out of RAM and stops.

`LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 865.91 M params.
Downloading: 100% 3.94G/3.94G [05:01<00:00, 13.1MB/s]
^C`

**Comment by TheLastBen** [comment]
Disconnect and reconnect and try again, sometimes it fails, sometimes not
